### DISCUSSION TITLE
Sample main.go on https://docs.pterm.sh/getting-started

### DISCUSSION THREAD

**Discussion by imattf** [pterm/pterm#372] [Q&A] [answered]
Hi Folks, found your posting on HackerNews... Great Job!!

I had to modify the sample code from: https://docs.pterm.sh/getting-started to get a good run...

```
//added reference to putils

import (
	"github.com/pterm/pterm"
	"github.com/pterm/pterm/putils" 
)
```

also line #24 changed to:
```
//couldn't find NewLettersFromString()

_ = pterm.DefaultBigText.WithLetters(putils.LettersFromString("Hello")).Render()
```

...is this headed in right direction?

**Comment by syntaqx** [comment]
Might be worth submitting a PR against https://github.com/pterm/docs incase they've drifted from current

**Comment by floaust** [comment, marked as answer]
@imattf you are absolutely right, `NewLettersFromString()` is not a thing and as you wrote it must be replaced with `LettersFromString()`.
Thanks for the feedback. 😃 

PS: will be fixed asap

**Comment by MarvinJWendt** [comment]
Nice catch! 
The fixed version is live. The old version might still be cached, so just hit `ctrl`+`shift`+`r` to force reload, if you still see the old version :)
